src/render/python/shape_v.cpp

```diff
@@ -301,6 +301,7 @@ MI_PY_EXPORT(Shape) {
         .def_method(Shape, id)
         .def_method(Shape, is_mesh)
         .def_method(Shape, parameters_grad_enabled)
+        .def_method(Shape, set_bsdf, "bsdf"_a.none())
```

Comment:
Do we support `nullptr` as BSDF? I suspect that this would crash the rendering process, at least in scalar mode. By default, a diffuse BRDF is created when none is specified.

Reply:
Good point. I added this thinking "might as well", but I didn't have a precise use-case in mind. I'll remove it for safety.